airflow/contrib/example_dags/example_gcp_sql.py

```diff
@@ -123,6 +139,45 @@
         task_id='sql_instance_patch_task'
     )
     # [END howto_operator_cloudsql_patch]
+    # [START howto_operator_cloudsql_db_insert]
+    sql_db_insert_task = CloudSqlInstanceDatabaseInsertOperator(
+        project_id=PROJECT_ID,
+        body=db_insert_body,
+        instance=INSTANCE_NAME,
+        task_id='sql_db_insert_task'
+    )
+    # [END howto_operator_cloudsql_db_insert]
+    # A second, identical database insert to verify idempotence
+    sql_db_insert_task_2 = CloudSqlInstanceDatabaseInsertOperator(
```

Comment:
You're quite right, the doubled operators' invocations in the example DAG may be confusing for users.
Moreover, I realised that idempotence in our cases is about not performing an operation in GCP under certain conditions (insert - if sth already exists, delete - if sth doesn't exist), so it can in fact be tested with unit tests I guess and that should be enough.

Therefore I'll remove the doubled operators from the DAG and I'll add any missing unit tests to cover this.

Reply:
Done, removed the doubled operators.